src/libs/actions/Workflow.ts

```diff
@@ -169,7 +170,13 @@ function updateApprovalWorkflow(policyID: string, approvalWorkflow: ApprovalWork
         },
     ];
 
-    const parameters: UpdateWorkspaceApprovalParams = {policyID, authToken, employees: JSON.stringify(Object.values(updatedEmployees))};
+    // const newDefaultApprover = approvalWorkflow.isDefault ? approvalWorkflow.approvers[0].email : undefined;
```

Comment:
@blazejkustra What problem here?

Reply:
This 😢 https://github.com/Expensify/App/issues/47344#issuecomment-2287895193